cmd/operator-sdk/test/local.go

```diff
@@ -82,6 +83,7 @@ func newTestLocalCmd() *cobra.Command {
 		"Use a different operator image from the one specified in the namespaced manifest")
 	testCmd.Flags().StringVar(&tlConfig.localOperatorFlags, "local-operator-flags", "",
 		"The flags that the operator needs (while using --up-local). Example: \"--flag1 value1 --flag2=value2\"")
+	testCmd.Flags().BoolVar(&tlConfig.skipCleanupOnError, "skip-cleanup-error", false, "The clean up function resposable to remove all artefacts will be skipped if an error be faced and it be set as true.")
```

Comment:
```suggestion
	testCmd.Flags().BoolVar(&tlConfig.skipCleanupOnError, "skip-cleanup-error", false, 
	"The clean up function resposable to remove all artefacts will be skipped if an error be faced and it be set as true.")
```

to solve lll.
Do not formet to run go fmt ./... to ensure indentation,